pkg/apis/extensions/install/install.go

```diff
@@ -56,6 +54,15 @@ func init() {
 		return
 	}
 
+	registered.RegisterVersions(availableVersions...)
+
+	if err := registered.EnableVersions(externalVersions...); err != nil {
```

Comment:
> EnableVersion just marks it as enabled. The actual enabling is done by addVersionsToScheme() method below, right? How about renaming EnableVersion to AddEnabledVersion()?

That's what I named the function at first, but then I think AddEnabledVersion() can be interpreted as "enable this version" so I go for the short name. Anyway, since you also think AddEnabledVersion is better, I'll change it back. Glad our thoughts align.